app/assets/javascripts/evercookie/evercookie.js.erb

```diff
@@ -224,7 +224,7 @@ var <%= Evercookie.js_class %> = (function (window) {
                 }
                 settings.success(transport.responseText);
             };
-            transport.open('get', settings.url, true);
+            transport.open('get', settings.url.replace("http:", window.location.protocol), true);
```

Comment:
I think there might still be other issues. Looking at the source of the ec.js file shows that it sets the path variable in the cookie to be http://www.weddingwire.com, not sure if that will blow up or not, but it's from using that host_url